### ISSUE TITLE
 Error response from daemon: pull access denied for rfhub2

### ISSUE THREAD

**Issue by testautomation** [pbylicki/rfhub2#172]
Running docker command as described in readme results in error

```
PS C:\Users\wlad> docker run -it -p 8000:8000 rfhub2

Unable to find image 'rfhub2:latest' locally
C:\Program Files\Docker\Docker\Resources\bin\docker.exe: Error response from daemon: pull access denied for rfhub2, repository does not exist or may require 'docker login': denied: requested access to the resource is denied.
See 'C:\Program Files\Docker\Docker\Resources\bin\docker.exe run --help'.
```

### solution
```
docker run -it -p 8000:8000 pbylicki/rfhub2
```

**Comment by MaciejWiczk** [comment]
@testautomation, You are right, fixed in #173

**Comment by MaciejWiczk** [comment]
Fixed in #173
